test/e2e/admission.go

```diff
@@ -220,4 +220,1103 @@ var _ = ginkgo.Describe("Job E2E Test: Test Admission service", func() {
 		_, err = ctx.kubeclient.CoreV1().Pods(ctx.namespace).Create(pod)
 		gomega.Expect(err.Error()).Should(gomega.ContainSubstring(`the podgroup phase is Pending`))
 	})
+
+	ginkgo.It("Job mutate check", func() {
+		ctx := initTestContext(options{})
+		defer cleanupTestContext(ctx)
+
+		var job v1alpha1.Job
+		jsonData := []byte(`{
```

Comment:
Why not define Job directly here?

Reply:
Applies to all cases

Reply:
black box test use json style input may be better?